pkg/client/client.go

```diff
@@ -472,9 +475,16 @@ func (c *immuClient) Login(ctx context.Context, user []byte, pass []byte) (*sche
 		User:     user,
 		Password: pass,
 	})
-
+	if err != nil {
+		return nil, errors.FromError(err)
+	}
 	c.Logger.Debugf("login finished in %s", time.Since(start))
 
+	err = c.Tkns.SetToken("defaultdb", result.Token)
+	if err != nil {
+		return nil, err
+	}
+
 	return result, errors.FromError(err)
```

Comment:
Here the `err == nil`, so we can simplify:

```suggestion
	return result, nil
```